myriad-scheduler/src/test/resources/myriad-config-test-default.yml

```diff
@@ -20,6 +20,7 @@ servedBinaryPath: /tmp/myriadBinary
 mesosMaster: 10.0.2.15:5050
 haEnabled:  false
 checkpoint: false
+isSecurityEnabled: false
```

Comment:
maybe add to src/main/resources/myriad-default-config for a minimal documentation.